src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineFunctions.scala

```diff
@@ -338,6 +372,14 @@ object PipelineFunctions {
       s"modules include ${pipelineModules.map(m => s"\n(${m.moduleId}, ${m.moduleName})").mkString("\n")}"))
   }
 
+  private[overwatch] def deriveSKU(isAutomated: Column, sparkVersion: Column): Column = {
+    val isJobsLight = sparkVersion.like("apache_spark_%")
+    when(isAutomated && isJobsLight, "jobsLight")
+      .when(isAutomated && !isJobsLight, "automated")
```

Comment:
just leave `isAutomated`, we already checked for `isJobsLight`

Reply:
fair enough, added to later branch. quicker / easier than fixing here due to pending commits and what not.